### PR TITLE
removing reference to headless DC

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Description
As per the TC meeting, headless DC will be shut down on **May 12**, thus need to remove any reference in the docs.
Update: it has been pushed out to the end of next week (May 19)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/secure-connections/sauce-connect/system-requirements.md [hunk 1]
```diff
@@ -302,7 +302,7 @@ When your tests are running through a Sauce Connect tunnel, the client on your n
 
 If you're testing in a restricted network setting, you may need to allowlist the Sauce Labs domains below to allow outbound communication to Sauce Labs Selenium and Appium endpoints. Allowlisting for inbound traffic coming into your network is not necessary. To confirm your setup is successful, try running a test using the [basic setup](/secure-connections/sauce-connect/setup-configuration/basic-setup).
 
-You'll need to use the set of domains for your corresponding Sauce Labs data center: US data center (US West 1), European data center (EU Central 1), headless data center (Headless US-East), or Australian data center (APAC Southeast 1). The data center you're connected to will be indicated in your navigation.<br/>
+You'll need to use the set of domains for your corresponding Sauce Labs data center: US data center (US West 1), European data center (EU Central 1), or Australian data center (APAC Southeast 1). The data center you're connected to will be indicated in your navigation.<br/>
```
Comment (LoryApiFortress) [inline docs/secure-connections/sauce-connect/system-requirements.md:305]:
```suggestion
You need to use the set of domains for your corresponding Sauce Labs data center: US data center (US West 1), European data center (EU Central 1), or Australian data center (APAC Southeast 1). The data center you're connected to is indicated in your navigation.<br/>
```

File: docs/secure-connections/sauce-connect/system-requirements.md [hunk 2]
```diff
@@ -317,8 +317,7 @@ groupId="dc-url"
 defaultValue="US-West-1"
 values={[
 {label: 'US-West-1', value: 'US-West-1'},
-{label: 'EU-Central-1', value: 'EU-Central-1'},
-{label: 'Headless US-East', value: 'Headless US-East'},
+{label: 'EU-Central-1', value: 'EU-Central-1'}
```
Comment (LoryApiFortress) [inline docs/secure-connections/sauce-connect/system-requirements.md:320]:
```suggestion
{label: 'EU-Central-1', value: 'EU-Central-1'},
```

**Review by aliciakrebs — APPROVED** [review]
Approved. NOTE: Hold merge until date is confirmed.

**Comment by LoryApiFortress** [comment]
Converted to Draft for the time being so that the Merge button is disabled to avoid accidentally merged

**Pull request merged**